src/nginx/t/transcoding_errors.t

```diff
@@ -137,7 +137,7 @@ $response = ApiManager::http_get($NginxPort,'/shelves/100?key=api-key-3');
 
 like($response, qr/HTTP\/1\.1 404 Not Found/, 'Got a 404.');
 like($response, qr/Content-Type: application\/json/i, 'Content-type is application/json');
-like($response, qr/Shelf not found/i, "the message from the backend was propogated.");
+like($response, qr/Cannot find shelf 100/i, "the message from the backend was propogated.");
```

Comment:
add comment that this error message is from detail_status from gRPC backend.

Reply:
Done.